<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>262</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Viewer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QGridLayout" name="gridLayout" rowstretch="4,0,0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <item row="2" column="6" colspan="2">
           <widget class="QRadioButton" name="radioButton_X">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="11">
           <widget class="QScrollBar" name="horizontalScrollBar_slice">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QSpinBox" name="spinBox_slice"/>
          </item>
          <item row="2" column="5">
           <widget class="QRadioButton" name="radioButton_Z">
            <property name="text">
             <string>Z</string>
            </property>
           </widget>
          </item>
          <item row="1" column="5" colspan="4">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Iterate over :</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" colspan="11">
           <widget class="QTextBrowser" name="Sample">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>40</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="Load">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
          <item row="2" column="8">
           <widget class="QRadioButton" name="radioButton_Y">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="channel_name">
            <property name="text">
             <string>Channel Name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Slice number</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="Qchannel_name"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Filestructure">
       <attribute name="title">
        <string>File structure</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>Object</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Dataset size</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1005</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_slice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>horizontalScrollBar_slice</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>721</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>185</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalScrollBar_slice</sender>
   <signal>valueChanged(int)</signal>
   <receiver>spinBox_slice</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>654</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>710</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
